ddd-kata.md

```diff
@@ -0,0 +1,94 @@
+# Domain-Driven Design Kata
+
+## What is a Kata?
+
+A kata is a specific sequence of movements and techniques practiced in the martial arts. The word "kata" is Japanese and is used in many martial arts originating from Japan, such as karate, judo, and aikido. Kata is usually taught by a sensei, who will demonstrate the movements and then guide the students as they practice the kata. The practice of kata can help martial artists to develop proper technique, balance, and control, and to learn the application of various techniques in a practical context. Some martial arts styles have kata that are performed solo, while others require a partner.
+Additionally, Kata is also commonly used in software development, to practice writing software that accomplish specific tasks, by following a predetermined set of steps. The goal of the practice is to create software that is reliable, easy to maintain and flexible. These principles are now applied in practicing Domain-Driven Design.
```

Comment:
```suggestion
Additionally, kata is also commonly used in software development to practice writing software that accomplishes specific tasks, by following a predetermined set of steps. The goal of the practice is to create software that is reliable, easy to maintain and flexible. These principles are now applied in practicing Domain-Driven Design.
```

Reply:
> Additionally, kata is also commonly used

I think there's a grammar issue here but I'm unsure what the correct sentence should be. From the way you used the term above, "kata" seems to also be the plural form (see also https://www.merriam-webster.com/dictionary/kata)? Then this sentence should either be "the term kata is also commonly used" or "kata are also commonly used". At least that's how I understand it.